src/ucp/core/ucp_ep.c

```diff
@@ -265,19 +265,20 @@ static void ucp_ep_destroy_base(ucp_ep_h ep)
 
 void ucp_ep_add_ref(ucp_ep_h ep)
 {
-    ucs_assert(ep->refcounter < UINT8_MAX);
-    ++ep->refcounter;
+    ucs_assert(ep->refcount < UINT8_MAX);
+    ++ep->refcount;
 }
 
+/* Return value indicates whether UCP EP was destroyed or not */
```

Comment:
/* Return 1 if was destroyed, 0 if not */
pls squash after fix

Reply:
fixed